### PR TITLE
Fix vSphere guide page to reflect necessary info

### PR DESCRIPTION
**Pull request by denismakogon**
Changes:
 - grammar
 - deployPkg guide included

### CODE REVIEW

File: source/guide/3.2/plugin-vsphere.md [hunk 1]
```diff
@@ -41,9 +41,10 @@ ssh-keygen -b2048 -N "" -q -f ~/.ssh/cloudify-agent-kp.pem
...
+* Both templates must have SSH activated and open on the firewall.
 * Both templates must have VMWare tools installed. Instructions for this can be found on the [VMWare site](http://kb.vmware.com/selfservice/microsites/search.do?language=en_US&cmd=displayKC&externalId=2075048). Please note, however, that the instructions on this site give incorrect tools for importing keys (it should be using `rpm --import <key>` rather than the apt-key equivalent). After following the instructions you should also run: `chkconfig vmtoolsd on`.
+* Also it is necessary to install deployPkg plugin into VM according to [VMWare documentation](http://kb.vmware.com/selfservice/microsites/search.do?language=en_US&cmd=displayKC&externalId=2075048)
```
Comment (ran-z) [inline source/guide/3.2/plugin-vsphere.md]: I think I'd rephrase this a bit into:
"It is also necessary to install the deployPkg plugin on the VM..."

I know it's nitpicking, but the PR is mostly about things like that so I figured I should point that out :)

Comment (denismakogon) [inline source/guide/3.2/plugin-vsphere.md]: @ran-z done, submitting.
